src/gantry_calibration/README.md

```diff
@@ -0,0 +1,88 @@
+# Gantry Calibration
+
+*By Andrew Lau*
+
+A Cartesian gantry or XY system serving a large area often requires calibration to achieve the
+accuracy required.
+
+This is a Python example showing how to take the expected and actual coordinates of a few grid
+points on the gantry, and generate a mapping from desired coordinates to calibrated coordinates.
+
+|         Basic Bilinear           |            Biquadratic              |             Real Example              |
+|:--------------------------------:|:-----------------------------------:|:-------------------------------------:|
+| ![basic bilinear](img/basic.png) | ![biquadratic](img/biquadratic.png) | ![real example](img/real_example.png) |
+
+- On the left, the basic premise is that the user commands the stage to move to the blue *expected* locations $(x_p, y_p)$, but finds that the red *actual* locations are at machine coordinates $(x_p', y_p')$. By recording the coordinates of both the blue and red dots, we can come up with a mapping function that takes the expected or raw machine coordinates (blue), and return the actual or calibrated coordinates (red) for anywhere in the area.
```

Comment:
This point confused me a bit because it is the opposite what is being described in calibration_map.md. Here it says that commanding $(x_p, y_p)$ results in the machine ending up $(x_p', y_p')$, while in calibration_map.md $(x_p', y_p')$ is the command that needs to be sent into order to move the machine to "hit the fiducial mark or sample" which is at $(x_p, y_p)$.

Reply:
Refactored both .md files for clarity.